lib/sparse/src/index/search_context.rs

```diff
@@ -19,7 +19,7 @@ pub struct IndexedPostingListIterator<'a> {
 }
 
 /// Making this larger makes the search faster but uses more (pooled) memory
-const ADVANCE_BATCH_SIZE: usize = 10_000;
+const ADVANCE_BATCH_SIZE: usize = 16_384;
```

Comment:
By the way, [`pow`](https://doc.rust-lang.org/std/primitive.u64.html#method.pow) is `const` these days, so we can even do:

```rust
const ADVANCE_BATCH_SIZE: usize = 2.pow(14);
```

Totally fine to leave it as is though, I don't suggest to change it. :+1: